odom_topic: "/auto_odom"

output_odom_topic: "/t2_odom"
new_frame_id: "new_velodyne"

# tform.T'
# LG57
t2: [0.999,0.030,-0.022,0.000,-0.029,0.999,0.014,0.000,0.023,-0.013,1.000,0.000,-1.594,2.370,-1.608,1.000]

lidar_topic: "/velodyne_points"
output_lidar_topic: "/ignore_velodyne_points"
# =========================== parameter's for point republish to remove the point inside angle and dis 
ignore_angle_front: 10
ignore_dis_front: 0.2
ignore_angle_rear: 40
ignore_dis_rear: 0.8